### PR TITLE
New feature method

### PR DESCRIPTION
**Pull request by Sgarc-027**
In this branch I have added the new method inside of the FeatureSelection class. The new method return a data frame of the best features and the target and a list of the features. I also created a script on how to access this method from FeatureSelection. In addition, the script also contains the score of using the best features in the Decision Tree Classifier and a heatmap of the correlations between the best features.

### CODE REVIEW

**Comment by tommy-waltmann** [comment]
Make sure you merge master into the `new_feature_method` branch. That will incorporate the changes from my recent PR which added the stuff for hyperparameter optimization

File: voting_ml/feature_selection.py [hunk 1]
```diff
@@ -358,5 +359,63 @@ def ftsel_KBest(self, X_train, y_train, X_test, K, score_fn=chi2):
...
+        for i in range(0, N): 
+            max1 = 0 
+            for j in range(len(list1)):
+                if list1[j] > max1:
+                    max1 = list1[j];
+
+            list1.remove(max1);
+            final_list.append(max1)
+
+        return final_list
```
Comment (tommy-waltmann) [inline voting_ml/feature_selection.py]: Does numpy have a function that can do this?

**Comment by Sgarc-027** [comment]
I think I could've done is sort the list first them do the following
list[-N:] where N is the number of features I want

**Comment by tommy-waltmann** [comment]
If you fix the above and merge master into this branch, this should be in good shape.

**Pull request merged**